### PR TITLE
planning: add interface to set vehicle signal

### PR DESCRIPTION
**Pull request by jmtao**

### CODE REVIEW

File: modules/planning/common/reference_line_info.cc [hunk 1]
```diff
@@ -592,15 +599,33 @@ void ReferenceLineInfo::ExportTurnSignal(VehicleSignal* signal) const {
...
+void ReferenceLineInfo::SetEmergencyLight() {
+  vehicle_signal_.set_emergency_light(true);
+}
+
+void ReferenceLineInfo::ExportVehicleSignal(
+    common::VehicleSignal* vehicle_signal) const {
+  CHECK_NOTNULL(vehicle_signal);
+  vehicle_signal->Clear();
+
+  vehicle_signal->CopyFrom(vehicle_signal_);
```
Comment (xiaoxq) [inline modules/planning/common/reference_line_info.cc]: Simply `*vehicle_signal = vehicle_signal_;`, so you don't need to clear.

File: modules/planning/scenarios/park/pull_over_emergency/BUILD [hunk 2]
```diff
@@ -6,9 +6,11 @@ cc_library(
     name = "pull_over_emergency_scenario",
     srcs = [
         "pull_over_emergency_scenario.cc",
+        "stage_approach.cc",
```
Comment (xiaoxq) [inline modules/planning/scenarios/park/pull_over_emergency/BUILD]: Prefer adding its own BUILD target.

File: modules/planning/scenarios/park/pull_over_emergency/stage_approach_test.cc [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#define protected public
+#define private public
```
Comment (xiaoxq) [inline modules/planning/scenarios/park/pull_over_emergency/stage_approach_test.cc]: Seems not necessary?

File: modules/planning/common/reference_line_info.cc (hunk 1 above)
Comment (jmtao) [reply modules/planning/common/reference_line_info.cc]: amended. thx!

File: modules/planning/scenarios/park/pull_over_emergency/BUILD (hunk 2 above)
Comment (jmtao) [reply modules/planning/scenarios/park/pull_over_emergency/BUILD]: there's circular dependency among the scenario and stages. 
will build them together as is.

File: modules/planning/scenarios/park/pull_over_emergency/stage_approach_test.cc (hunk 3 above)
Comment (jmtao) [reply modules/planning/scenarios/park/pull_over_emergency/stage_approach_test.cc]: keep it for future use. 
we may add more tests later.

**Pull request merged**